src/test/java/org/vafer/jdependency/ClazzpathUnitTestCase.java

```diff
@@ -153,13 +167,13 @@ public void testShouldHaveUnitId() throws IOException {
         final Clazzpath cp = new Clazzpath();
 
         final ClazzpathUnit u1 = cp.addClazzpathUnit(resourceFile("jar1.jar"));
-        assertEquals(u1.toString(), "jar1.jar");
+        assertTrue(u1.toString().endsWith("jar1.jar"));
 
         final ClazzpathUnit u1e = cp.addClazzpathUnit(resourceFile("jar1.jar"), "jar1");
         assertEquals(u1e.toString(), "jar1");
 
         final ClazzpathUnit u2 = cp.addClazzpathUnit(resourcePath("jar2.jar"));
-        assertEquals(u2.toString(), "jar2.jar");
+        assertTrue(u2.toString().endsWith("jar2.jar"));
```

Comment:
Some here. Could/should we be more specific than the `endsWith`

Reply:
Yes, perhaps it can be a bit more specific

Reply:
The actual value on my machine is `/home/nbasjes/workspace/Apache/jdependency/target/test-classes/jar1.jar` so I think testing for endsWith `/jar1.jar` is the right way.

Reply:
Updated